pkgs/development/python-modules/google-auth-oauthlib/default.nix

```diff
@@ -28,6 +29,8 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  disabledTests = lib.optionals stdenv.isDarwin [ "test_run_local_server" ];
```

Comment:
Did you try ``__darwinAllowLocalNetworking = true;``?

Reply:
you asked this above. I have not heard of this option. Can you please provide a more specific recommendation about how to use it. Does it go into this derivation?

Reply:
`__darwinAllowLocalNetworking = true;`  works so I removed the manually disabled commits.